pkg/webui/lib/components/message/index.js

```diff
@@ -70,6 +74,13 @@ const Message = function({
     return renderContent(content, component, rest)
   }
 
+  if (hasMarkdown) {
+    const contentWithMarkdown = formatMessage(content, vals)
+    return reactStringReplace(contentWithMarkdown, /`([^`[]+)`/g, (match, i) => (
+      <code key={i}>{match}</code>
+    ))
+  }
```

Comment:
```suggestion
  if (hasMarkdown) {
    const contentWithMarkdown = formatMessage(content, vals)
    return renderContent(reactStringReplace(contentWithMarkdown, /`([^`[]+)`/g, (match, i) => (
      <code key={i}>{match}</code>
    )), component, rest)
  }
```

This should work too, no? Otherwise, it's not possible to use the `component` or `className` prop.

Reply:
Good catch, thanks!